plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/KuduSmokeTests.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kudu;
+
+public class KuduSmokeTests
+{
+    private static final String KUDU_VERSION = "1.13.0";
+
+    public static class TestKuduSmokeTestWithDisabledInferSchema
```

Comment:
Any reason this is not defined as a top-level class?

Reply:
See https://github.com/trinodb/trino/pull/10940#discussion_r806479586. I believe to keep all tests for same version together.

Reply:
Sharing a constant doesn't require nesting classes.
i admire cleverness, but this also means unnecessary class hierarchy, which doesn't help browse the code.

if the paradigm was more frequent in the code base, i would probably get used to it and wouldn't complain.

Reply:
I noticed depending on how you run the test `mvn test -Dtest=io.trino.plugin.kudu.KuduLatestConnectorTests` vs `mvn test` some tests will be skipped in the class hierarchy. 

Additionally, when trying to run tests through intellij's UI you can only run the static inner classes (not the top level class).

Seems like the tooling support for static inner test classes is just not good, I'm going to move these to top level classes